cypress/integration/dib-travel/company-settings/travel-policy.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { TravelPolicy } from '../../../models';
+
+describe('Travel Policy Suite', () => {
+  let tpd: TravelPolicy;
+
+  before(() => {
+    cy.fixture('company-settings/travel-policy').then((travelPolicyFormData) => {
+      tpd = travelPolicyFormData;
+    });
+  });
+
+  beforeEach(() => {
+    cy.login();
+    cy.get('dib-layout dib-hamburger-icon').click();
+    cy.get('[href="/company-management/travel-policy"]').click();
+  });
+
+  it('submits empty travel policy form', () => {
+    cy.get('dib-travel-policy ui-button[type=primary]').click();
+    cy.get('dib-travel-policy-dialog ui-button button').click({ force: true });
```

Comment:
Revisit why you need `{ force: true }`? Try a different selector or something different.